sdk/networkanalytics/arm-networkanalytics/review/arm-networkanalytics.api.md

```diff
@@ -311,15 +258,15 @@ export interface DataProductUpdateProperties {
 }
 
 // @public
-export type DataProductUserRole = string;
+export type DataProductUserRole = string | "Reader" | "SensitiveReader";
```

Comment:
why we have this gap?

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/networkanalytics/NetworkAnalytics.Management/main.tsp#L423-L436

Reply:
current code in swagger https://github.com/Azure/azure-rest-api-specs/blob/main/specification/networkanalytics/resource-manager/Microsoft.NetworkAnalytics/stable/2023-11-15/networkanalytics.json
current code in sdk:
```
/** Known values of {@link DataProductUserRole} that the service accepts. */
export enum KnownDataProductUserRole {
  /** Field to specify user of type Reader. */
  Reader = "Reader",
  /**
   * Field to specify user of type SensitiveReader.
   * This user has privileged access to read sensitive data of a data product.
   */
  SensitiveReader = "SensitiveReader"
}

/**
 * Defines values for DataProductUserRole. \
 * {@link KnownDataProductUserRole} can be used interchangeably with DataProductUserRole,
 *  this enum contains the known values that the service supports.
 * ### Known values supported by the service
 * **Reader**: Field to specify user of type Reader. \
 * **SensitiveReader**: Field to specify user of type SensitiveReader.
 * This user has privileged access to read sensitive data of a data product.
 */
export type DataProductUserRole = string;
```